projects/js-packages/publicize-components/src/components/form/use-connection-state.ts

```diff
@@ -52,6 +52,7 @@ export const useConnectionState = () => {
 
 			// 3. Not have a NO_MEDIA_ERROR when media is required
 			const hasNoMediaError = validationErrors[ currentId ] === NO_MEDIA_ERROR;
+			return true;
```

Comment:
This is added until the frontend validation is figured out to be able to share to Instagram without an image selected